server/src/main/java/org/apache/uniffle/server/ShuffleTaskManager.java

```diff
@@ -165,8 +172,21 @@ public ShuffleTaskManager(
             .maximumSize(Integer.MAX_VALUE)
             .build();
 
+    startClearResourceThread();
+    memoryExporter =
```

Comment:
`memoryExporter` is not accurate. Please rename this.

Reply:
ok, I rename it to: topNShuffleDataSizeOfAppCalcTask